src/lib/setup.js

```diff
@@ -235,6 +235,12 @@ export default function(app) {
     app.use(splitTestMiddleware)
   }
 
+  // Maybe the url wants to display a banner
+  app.use((req, res, next) => {
+    res.locals.sd.flashMessage = req.query.flashMessage
+    next()
+  })
```

Comment:
Is this the best place to do this? Feels hacky.

Reply:
I think this would fit best as part of our [existing middleware](https://github.com/artsy/force/blob/master/src/lib/middleware/locals.coffee), but the general approach makes sense!